core/common/src/test/java/alluxio/AlluxioTestDirectory.java

```diff
@@ -90,12 +81,19 @@ private static void cleanUpOldFiles(File dir) {
     File[] files = dir.listFiles();
     for (File file : files) {
       if (!FileUtils.isFileNewer(file, cutoffTimestamp)) {
-        try {
-          alluxio.util.io.FileUtils.deletePathRecursively(file.getAbsolutePath());
-        } catch (Exception e) {
-          LOG.warn("Failed to delete {} : {}", file.getAbsolutePath(), e.getMessage());
-        }
+        delete(file);
       }
     }
   }
+
+  private static void delete(File file) {
+    if (!file.exists()) {
+      return;
+    }
+    try {
+      alluxio.util.io.FileUtils.deletePathRecursively(file.getAbsolutePath());
+    } catch (IOException e) {
+      LOG.warn("Failed to clean up {} : {}", file.getAbsolutePath(), e.toString());
```

Comment:
`e.getMessage`?

Reply:
When a file is unreadable, delete throws an exception along the lines of `new MissingPermissionsException(filename)`. Using `e.getMessage()` we lose the fact that we failed because of permissions, and only get the filename.

Reply:
This seems to be a very case-by-case decision, since I think some exceptions have a very cryptic `toString`, is there a common denominator we could normalize on? I think printing the whole exception (w/ stack trace) is fairly expensive, perhaps some combination of the two in a utils method?